frameworks/cassandra/tests/test_racks.py

```diff
@@ -30,21 +29,31 @@ def test_rack():
         config.get_foldered_service_name(),
         3,
         additional_options={
-            "service": {
-                "name": config.get_foldered_service_name()
-            },
-            "nodes": {
-                "placement_constraint": "[[\"@zone\", \"GROUP_BY\", \"1\"]]"
-            }
-        })
-
-    raw_status = nodetool.cmd('node-0', 'status')
+            "service": {"name": config.get_foldered_service_name()},
+            "nodes": {"placement_constraint": '[["@zone", "GROUP_BY", "1"]]'},
+        },
+    )
+
+    raw_status = nodetool.cmd("node-0-server", "status")
```

Comment:
Is the change to `node-0-server` necessary for sdk-0.40? I took a quick look didn't see build failures related to it.

Reply:
Not strictly. It is a bit more specific though. I can remove it tomorrow.

Reply:
Did it affect the test? https://teamcity.mesosphere.io/viewLog.html?buildId=1134401&tab=buildResultsDiv&buildTypeId=DcosIo_DcosCommons_Cassandra_BuildAndTestPr#testNameId7370747642665586932

Reply:
🤦‍♂️ in this branch `nodetool.cmd` *already* appends `-server`. Fixed and force-pushed.

Reply:
ping @mpereira.